docs/visualization/path/response_time.md

```diff
@@ -74,15 +74,31 @@ plot.show()
 # plot.show()
 ```
 
-![response_time_worst_stacked_bar](../../imgs/response_time_worst_stacked_bar.png)
+![response_time_worst_stacked_bar](../imgs/response_time_stacked_bar_worst.png)
 
 ```python
 # plot best case
 plot = Plot.create_response_time_stacked_bar_plot(path, case='best')
 plot.show()
 ```
 
-![response_time_best_stacked_bar](../../imgs/response_time_best_stacked_bar.png)
+![response_time_best_stacked_bar](../imgs/response_time_stacked_bar_best.png)
+
+```python
+# plot best case
```

Comment:
```suggestion
# plot worst-in-input case
```

Reply:
Fixed it in [fb0f2c2](https://github.com/tier4/CARET_doc/pull/242/commits/fb0f2c266f386760dec191d05ed5980264018e89).